orangecontrib/network/network/base.py

```diff
@@ -134,6 +134,16 @@ def __init__(self,
         self.twoway_edges = self.edges + self.edges.transpose()
         self.twoway_edges.sum_duplicates()
 
+        # Manually fill in back the zero weights that get removed by summing sparse matrices
+        for idx_row in range(self.edges.shape[0]):
+            col_indices = self.edges.indices[self.edges.indptr[idx_row]:
+                                             self.edges.indptr[idx_row + 1]]
+
+            zero_data = self.edges[idx_row, col_indices].toarray().flatten() == 0
+            col_indices = col_indices[zero_data]
+            self.twoway_edges[idx_row, col_indices] = 0.0
+            self.twoway_edges[col_indices, idx_row] = 0.0
+
```

Comment:
This way of manipulating a sparse matrix is apparently not very efficient (a warning pops up). I haven't worked with scipy sparse matrices up till now, so I'd appreciate if someone could suggest a better alternative (if it exists) :slightly_smiling_face: